test-mocha/panels/lovelace/editor/config-util.spec.ts

```diff
@@ -52,3 +55,55 @@ describe("swapCard", () => {
     assert.deepEqual(expected, result);
   });
 });
+
+describe("moveCard", () => {
+  it("move a card to an empty view", () => {
+    const config: LovelaceConfig = {
+      views: [
+        {},
+        {
+          cards: [{ type: "card1" }, { type: "card2" }],
+        },
+      ],
+    };
+
+    const result = moveCard(config, [1, 0], [0]);
+    const expected = {
+      views: [
+        {
+          cards: [{ type: "card1" }],
+        },
+        {
+          cards: [{ type: "card2" }],
+        },
+      ],
+    };
+    assert.deepEqual(expected, result);
+  });
+
+  it("move a card to different view", () => {
```

Comment:
Can you add a test to move the card to the same view ? Although it's stupid to do, we should either raise an error or just work. (right now it doesn't work and you duplicate the card)